android/src/main/java/com/imagepicker/utils/MediaUtils.java

```diff
@@ -157,8 +149,7 @@
         ByteArrayOutputStream bytes = new ByteArrayOutputStream();
         scaledPhoto.compress(Bitmap.CompressFormat.JPEG, result.quality, bytes);
 
-        final boolean forceLocal = requestCode == REQUEST_LAUNCH_IMAGE_CAPTURE;
```

Comment:
This should probably be brought back as a local variable and ORed with the passed in version:
`final boolean finalForceLocal = (requestCode != REQUEST_LAUNCH_IMAGE_CAPTURE) || forceLocal;
final File resized = createNewFile(context, options, finalForceLocal);`
Otherwise this could break existing functionality of the module.